AEXS/aex-4.md

```diff
@@ -48,16 +48,29 @@ This is especially beneficial for mobile wallet users as they can simply click o
 
 2. Payment
 
-    URI: `aeternity:<receiver_address>/<amount>/<vendor_identifier>?callback=<url>&txId=<transaction_id>`
+    URI: `aeternity:<receiver_address>/<amount>/<vendor_identifier>?callback=<url>`
 
     This URI scheme enables vendors to generate direct payment request URIs to user wallet.
 
     1. Vendor generates a address for receiving payment.
-    2. Creates the URI substituting `receiver_address` with newly created address, `amount` with amount requested, `vendor_identifier` with a human readable identifier that can be used to correctly identifiy vendor and requested payment, and `url` with a callback URL that accepts a transaction id under `txId` query param.
+    2. Creates the URI substituting `receiver_address` with newly created address, `amount` with amount requested, `vendor_identifier` with a human readable identifier that can be used to correctly identify vendor and requested payment, and `url` with a callback URL that accepts a transaction id under `txId` query param.
```

Comment:
Is the `txId` still relevant here? Or should that mean that the callback url can contain that query parameter?

Reply:
I think it is 'cause the callback should accept the transaction id and it should be in a form of `txId` query param.